SPECS/xapi.spec

```diff
@@ -28,6 +29,7 @@ Patch1001: xapi-1.249.3-open-vxlan-port-for-sdn-controller.XCP-ng.patch
 Patch1002: xapi-1.249.3-create-plugged-vif-and-vbd-and-suspended-vm.XCP-ng.patch
 Patch1003: xapi-1.249.3-open-openflow-port.XCP-ng.patch
 Patch1004: xapi-1.249.3-update-db-tunnel-protocol-from-other_config.XCP-ng.patch
+Patch1005: xapi-1.249.3-expose-host-xen-scheduler-granularity-in-xapi.XCP-ng.patch
```

Comment:
If it was merged upstream (I think it was), please add a `Patch merged in xen-api 1.xxx.y` comment.

Reply:
Not yet, still in review.